### ISSUE TITLE
**Attention Security Professionals: Important Notice Regarding Sails-AdminPanel Security**

### ISSUE THREAD

**Issue by xziy** [sails-adminpanel/sails-adminpanel#13]
Greetings,

We appreciate your dedication to the security of the Sails-AdminPanel project. Before you proceed with reporting any potential security issues, we kindly request you to consult the SECURITY.md document associated with the project. In particular, we urge you to focus on the following points:

If, by any chance, you grant end-users unrestricted access to the EJS render method within the Sails-AdminPanel, it's important to recognize that such an approach inherently introduces security vulnerabilities. We kindly request that you avoid reporting security concerns arising from scenarios where this practice is employed.

EJS within Sails-AdminPanel functions as a dynamic JavaScript runtime. Its core purpose revolves around executing JavaScript code. However, if the EJS render method is employed without diligent input validation, the responsibility for the resulting outcomes lies with the implementer.

In simple terms, we kindly ask you not to submit security reports that revolve around the following code snippet:

```javascript
app.get('/', (req, res) => {
    res.render('index', req.query);
});
```

Your expertise and insights are highly valued. If you encounter situations where these considerations could enhance the security of the Sails-AdminPanel, we invite you to share your feedback through the admin panel. Your collaborative efforts play a pivotal role in maintaining the project's security standards.

Thank you for your commitment to the security of Sails-AdminPanel.
